src/sentry/api/endpoints/organization_details.py

```diff
@@ -570,28 +580,254 @@ def post_org_pending_deletion(
+        required=False,
+    )
+    attachmentsRole = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        help_text="The role required to download event attachments, such as native crash reports or log files.",
+        required=False,
+    )
+    debugFilesRole = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        help_text="The role required tto download debug information files, proguard mappings and source maps.",
+        required=False,
+    )
+
+    # avatar
+    avatarType = serializers.ChoiceField(
+        choices=(("letter_avatar", "Use initials"), ("upload", "Upload an image")),
+        help_text="The type of display picture for the organization",
+        required=False,
+    )
+    avatar = serializers.CharField(
+        help_text="The image to upload as the organization avatar, in base64. Required if `avatarType` is `upload`.",
+        required=False,
+    )
+
+    # security & privacy
+    require2FA = serializers.BooleanField(
+        help_text="Specify `true` to require and enforce two-factor authentication for all members.",
+        required=False,
+    )
+    allowSharedIssues = serializers.BooleanField(
+        help_text="Specify `true` to allow sharing of limited details on issues to anonymous users.",
+        required=False,
+    )
+    enhancedPrivacy = serializers.BooleanField(
+        help_text="Specify `true` to enable enhanced privacy controls to limit personally identifiable information (PII) as well as source code in things like notifications.",
+        required=False,
+    )
+    scrapeJavaScript = serializers.BooleanField(
+        help_text="Specify `true` to allow Sentry to scrape missing JavaScript source context when possible.",
+        required=False,
+    )
+    storeCrashReports = serializers.ChoiceField(
+        choices=(
+            (0, "Disabled"),
+            (1, "1 per issue"),
+            (5, "5 per issue"),
+            (10, "10 per issue"),
+            (20, "20 per issue"),
+            (50, "50 per issue"),
+            (100, "100 per issue"),
+            (-1, "Unlimited"),
+        ),
+        help_text="How many native crash reports (such as Minidumps for improved processing and download in issue details) to store per issue.",
+        required=False,
+    )
+    allowJoinRequests = serializers.BooleanField(
+        help_text="Specify `true` to allow users to request to join your organization.",
+        required=False,
+    )
+
+    # data scrubbing
+    dataScrubber = serializers.BooleanField(
+        help_text="Specify `true` to require server-side data scrubbing be enabled for all projects.",
+        required=False,
+    )
+    dataScrubberDefaults = serializers.BooleanField(
+        help_text="Specify `true` to require the default scrubbers be applied to prevent things like passwords and credit cards from being stored for all projects.",
```

Comment:
```suggestion
        help_text="Specify `true` to apply the default scrubbers to prevent things like passwords and credit cards from being stored for all projects.",
```